notebooks/tutorial_ivf_pq.ipynb

```diff
@@ -0,0 +1,1226 @@
+{
```

Comment:
<blockquote>Luckily training raft's ivf-pq does not take too much time thanks to using its fast implementation of Balanced Hierarchical K-Means clustering during build and an efficient encoding kernel. </blockquote><p>Maybe less sales pitchy to say something like:</p><p>"Since RAFT's IVF-PQ algorithm uses balanced-hierarchical k-means clustering and efficient logic for encoding, we find significantly improved index build times".</p><p> </p><blockquote> - <code>add_data_on_build</code> is a convenience flag that tells if you prefer to do both training (clustering) and filling the DB in one call to <code>ivf_pq.build</code> or start with an empty DB and add all data using <code>ivf_pq.extend</code> </blockquote><p>"when activated, automatically adds the training data to the index during <code>ivf_pq.build. </code>Otherwise, no data is added during <code>ivf_pq.build</code> and vectors need to be explicitly added to the index using  <code>ivf_pq.extend</code> "</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/raft/pull/1544/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>